monkestation/code/game/objects/items/granters/maintenanceencyclopedia.dm

```diff
@@ -0,0 +1,20 @@
+/obj/item/book/granter/crafting_recipe/maintgodgranter
+	name = "maintenance encyclopedia"
+	icon_state = "book1"
+	desc = "A burnt and damaged tome? Where did this come from?"
+	crafting_recipe_types = list(
```

Comment:
I think instead of just giving all of them it should pick 1 you dont already have to randomly give

Reply:
What would the best way to do this? Are there any examples of this from other granters?

Reply:
something like uh
var/static/list/possible_recipes = [this list here]
and then on init do 
`crafting_recipe_types = list(pick(possible_recipes))`